packages/design/src/colors.css

```diff
@@ -110,8 +110,10 @@
   --color-grey--lightest: rgb(244, 244, 244);
   --color-grey--dark: rgb(118, 118, 118);
 
-  --color-greyBlue--rgb: 101, 120, 132;
-  --color-greyBlue: rgb(var(--color-greyBlue--rgb));
+  /* TODO: Reintroduce RGB once the issue with JobberStyle is fixed */
+  /* --color-greyBlue--rgb: 101, 120, 132; */
+  /* --color-greyBlue: rgb(var(--color-greyBlue--rgb)); */
```

Comment:
Just a minor thing: wondering what the protocol is for comments like this in the code base since Git has this history :)

Reply:
Our plan is to revisit this this sprint for cleanup after we've got the ReactNative consumption issue unblocked, so I think it's OK to leave this kind of note for ourselves.